src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1651,17 +1784,20 @@ bool processOnChangeCondition
         if (condValues != NULL) {
             /* Check if some of the attributes in the NotifyCondition values list are in the entity.
              * Note that in this case we do a query for all the attributes, not restricted to attrV */
-            ContextElementResponseVector allCerV;
+            ContextElementResponseVector rawCerV, allCerV;
```

Comment:
no, I don't think it does.
We have a lot of style rules that are not detected by the script.
The style check script is in python and you are more than welcome to add to it ... :-)
I would do it but my python knowledge is below zero :-(